src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12198,8 +12198,8 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state, c
         }
     }
 
-    this->TotalCoolingCapacity = TotalCondCoolingCapacity * CoolingPLR;
-    this->TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR;
+    this->TotalCoolingCapacity = TotalCondCoolingCapacity * CoolingPLR * CyclingRatio;
+    this->TotalHeatingCapacity = TotalCondHeatingCapacity * HeatingPLR * CyclingRatio;
```

Comment:
Crankcase heater seems to have considered cycling
```
        VRFRTF = min(1.0, (CyclingRatio / PartLoadFraction));
...
        this->CrankCaseHeaterPower = this->CCHeaterPower * (1.0 - VRFRTF);
```
Defrost seems to have its own "runtime fraction", the `FractionalDefrostTime`: 
```
                    this->DefrostPower = DefrostEIRTempModFac * (this->HeatingCapacity / 1.01667) * FractionalDefrostTime;

                } else { // Defrost strategy is resistive
                    this->DefrostPower = this->DefrostCapacity * FractionalDefrostTime;
                }
```

Reply:
Crankcase heater power looks good. I think defrost power needs a  CyclingRatio term. FractionalDefrostTime is the defrost time for a full time step. If the system doesn't run that time step (i.e., CyclingRatio = 0.000000001) then DefrostPower should be 0. You can check FractionalDefrostTime and see that it's a function of OA humidity ratio and has nothing to do with PLR or CyclingRatio. Of course you have to check my thinking before doing anything.

Reply:
I think you're right. FractionalDefrostTime is either from the user input or from `OutdoorCoildw`. It should probably multiply by cycling ratio.

For `this->EvapCondPumpElecPower` term, since the FluidTCtrl model doesn't have the "Evaporative Condenser Pump Rated Power Consumption" field like the curve-based VRF, this term is always 0.